docs/content/home.md

```diff
@@ -12,43 +12,44 @@ menu:
 Home
 ====
 
-Here is some C# code
+MySqlConnector is an [ADO.NET](https://msdn.microsoft.com/en-us/library/e80y5yhx.aspx) data
+provider for [MySQL](https://www.mysql.com/). It provides implementations of
+`DbConnection`, `DbCommand`, `DbDataReader`, `DbTransaction` &ndash; the classes
+needed to query and update databases from managed code.  It's features include
 
-```csharp
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
-using MySqlConnector.Performance.Models;
+* .NET Core Compatibility
+* Truly Asynchronous: async functions implement asynchronous I/O
+* High Perforrmance: code is stress tested for performance bottlenecks
+* Ligtweight: Library only implements ADO.NET core
 
-namespace MySqlConnector.Performance.Controllers
-{
-    [Route("api/[controller]")]
-    public class AsyncController : Controller
-    {
-        // GET api/async
-        [HttpGet]
-        public async Task<IActionResult> Get()
-        {
-            using (var db = new AppDb())
-            {
-                await db.OpenAsync();
-                var cmd = BlogPost.LatestPosts(db);
-                var reader = await cmd.ExecuteReaderAsync();
-                var result = await BlogPost.ReadAllAsync(reader);
-                return new OkObjectResult(result);
-            }
-        }
+### Why use MySql over Oracle's Connector/Net?
 
-        // POST api/async
-        [HttpPost]
-        public async Task<IActionResult> Post([FromBody]BlogPost body)
-        {
-            using (var db = new AppDb())
-            {
-                await db.OpenAsync();
-                await BlogPost.InsertAsync(db, body);
-                return new OkObjectResult(body);
-            }
-        }
-    }
-}
-```
\ No newline at end of file
+MySqlConnector is a clean-room reimplementation of the [MySQL Protocol](https://dev.mysql.com/doc/internals/en/client-server-protocol.html)
+and is not based on [Oracle's Connector/Net](https://github.com/mysql/mysql-connector-net).
+
+<table class="table table-striped table-hover" style="max-width: 600px">
+  <thead>
+    <th style="width:25%"></th>
+    <th style="width:25%">MySqlConnector</th>
+    <th style="width:25%">Oracle's Connector/Net</th>
+    <th style="width:25%">MySqlConnector Advantage</th>
+  </thead>
+  <tr>
+    <td><strong>Async</strong></td>
+    <td>Fully Asynchronous I/O</td>
+    <td>Async calls map to synchronous I/O</td>
+    <td>Uses less thread pool threads</td>
+  </tr>
+  <tr>
+    <td><strong>Development</strong></td>
+    <td>Developed on <a href="https://github.com/mysql/mysql-connector-net">GitHub</a></td>
+    <td>Not developed in the open</td>
```

Comment:
Right, it does look like a mirror (all commits have full names, not GitHub usernames, which means the email addresses aren't associated with GitHub accounts).

I do agree that it feels like "open source" but not "open development". Phil Haack had a couple of posts on this a while ago:

* http://haacked.com/archive/2012/02/16/open-source-and-open-source-software-are-not-the-same.aspx/
* http://haacked.com/archive/2012/02/22/spirit-of-open-source.aspx/

In the latter, he used the phrase "open and collaborative development". It could work to put that in the MySqlConnector column and leave "Not developed in the open" here; that would draw a good contrast.